### DISCUSSION TITLE
Rework around setup, options, parameters and cookies

### DISCUSSION THREAD

**Discussion by mbab** [DivinumOfficium/divinum-officium#2825] [Developers corner]
I'm asking You check my work at https://github.com/mbab/divinum-officium/tree/hsetset
It makes code shorter and cleaner (at least to me), and make options in missa work.
I'm aware about error in missa cookie general (when moving from current) but changing cookie name will provide inconsistency to horas. 

If accepted, it will be followed by changes which make use of data in .dialog files in all selects.

**Comment by ajmalton** [comment]
I'm looking at it but I won't be able to do a full read.It looks good and the direction is right in my view.AJMOn 30 Mar 2022 05:19, mbab ***@***.***> wrote:
I'm asking You check my work at https://github.com/mbab/divinum-officium/tree/hsetset
It makes code shorter and cleaner (at least to me), and make options in missa work.
I'm aware about error in missa cookie general (when moving from current) but changing cookie name will provide inconsistency to horas.
If accepted, it will be followed by changes which make use of data in .dialog files in all selects.
